data-es2016plus.js

```diff
@@ -175,6 +175,7 @@ exports.tests = [
           chrome47:        true,
           edge14:          true,
           firefox43:       true,
+          safaritp:        true,
```

Comment:
@zloirock noticed you've point to wrong line. mistakenly added was `Array.prototype.includes`. This one (`%TypedArray%.prototype.includes`) was added in TP 9